cleverdiff/cleverdiff.py

```diff
@@ -38,13 +38,13 @@ def summarise_results(diffseen, diffresult):
     for seenindex, seenitem in enumerate(diffseen):
         result += "--------------------------------------------\n"
         result += f"DIFF {seenindex:3d}:\n" f"{str(seenitem)}"
-        result2 = {key: [] for key in MODE_DESCRIPTION.keys()}
+        result2 = {key: [] for key in MODE_DESCRIPTION}
```

Comment:
Thanks, this makes the code a bit simpler and more readable. However, as this is unrelated to the command-line argument changes, I'd prefer it to be in a separate pull request, along with the related change on line 47.